### PR TITLE
Adding trackingDatum fetcher

### PR DESCRIPTION
**Pull request by waldemarnt**
We just added a request that will get the last position of each equipment and will merge this data inside the equipment request body.
We added [https://lodash.com/](lodash) as a dependency that will support to navigate between arrays and extract values.

### CODE REVIEW

File: fetcher/canVariablesFetcher.js [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
 const defaultRequestOptions = {
   method: 'GET',
   json: true
 };
 
 const defaultCanVariableNames = [
-'ENGINE_HOURS',
-'ENGINE_SPEED',
-'DRIVING_DIRECTION',
+  'ENGINE_HOURS',
```
Comment (bltavares) [inline fetcher/canVariablesFetcher.js]: :+1:

File: fetcher/canVariablesFetcher.js [hunk 2]
```diff
@@ -44,18 +56,36 @@ class CanVariablesFetcher {
         Authorization: authorizationBearer
       }
     }, defaultRequestOptions);
-    
+
```
Comment (bltavares) [inline fetcher/canVariablesFetcher.js]: :+1:

**Comment by bltavares** [comment]
:+1:

**Comment by andressa** [comment]
👍

File: test/server_spec.js [hunk 3]
```diff
@@ -86,7 +126,7 @@ describe('EquipmentController', () => {
       const expectedResponse = {
         data: [
           generateFacadeEquipment('a-equipment-id-1'),
```
Comment (glauco) [inline test/server_spec.js]: Could we extract it into a variable called `equipmentOne`? Just for consistency.

File: test/server_spec.js [hunk 4]
```diff
@@ -127,18 +175,40 @@ describe('EquipmentController', () => {
...
+              116
+            ],
+            "type": "Point"
+          },
+          "status": "STOPPEDIDLE"
+        };
+    
       const expectedResponse = {
         data: [
           generateFacadeEquipment('a-equipment-id-1'),
```
Comment (glauco) [inline test/server_spec.js]: Could we extract it into a variable called equipmentOne? Just for consistency.

File: test/server_spec.js (hunk 3 above)
Comment (waldemarnt) [inline test/server_spec.js]: 👍

File: test/server_spec.js (hunk 4 above)
Comment (waldemarnt) [inline test/server_spec.js]: 👍

**Comment by glauco** [comment]
🚢 🇮🇹

**Pull request merged**